src/components/share/index.js

```diff
@@ -219,11 +341,17 @@ module.exports = {
         }
 	},
 	    setTypeAhead: function() {
-            let followernames = this.followernames;
+            let allNames = this.followernames.slice();
+            for(var i = 0; i < this.friendnames.length;i ++) {
+                let friend = this.friendnames[i];
+                if (allNames.indexOf(friend) == -1) {
```

Comment:
aren't all the friends in the followers?

Reply:
No. 
They are kept separate when derived in filsystem.js
followerNames = followerNames.filter(x => !friendNames.includes(x));